backend/src/main/java/com/huellapositiva/application/dto/ESALRequestDto.java

```diff
@@ -5,16 +5,78 @@
 import lombok.Builder;
 import lombok.Data;
 
-import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.Pattern;
 
 @Data
 @Builder
 @AllArgsConstructor
 public class ESALRequestDto {
 
     @Schema(
+            description = "Name of ESAL",
             example = "Canarias para todos"
     )
-    @NotEmpty
+    @NotBlank
+    @Pattern(regexp = "^[a-zA-Z ]+$")
     private final String name;
+
+    @Schema(
+            description = "ESAL's description",
+            example = "ESAL dedicada a ayudar en la recogida de alimentos."
+
+    )
+    private final String description;
+
+    @Schema(
+            description = "ESAL's webpage",
+            example = "https://www.foo-bar.com/"
+    )
+    @Pattern(regexp = "^https?://(www.)?[-a-zA-Z0-9+&@#%=~_|]*.(com|es|net|org)/[-a-zA-Z0-9+&@#%=~_|]*")
```

Comment:
This pattern is not correct in many ways.

@_NotEmpty is enough.
If you want to validate if a URL is valid and it exists, you could try making an HTTP HEAD request and verify that it returns a valid HTTP code and the content-type header is text/html

Reply:
Ok, i left only @_NotEmpty